src/Discord.Net.Core/Entities/Interactions/Modals/ModalBuilder.cs

```diff
@@ -108,21 +110,109 @@ public ModalBuilder AddComponents(List<IMessageComponent> components, int row)
             return this;
         }
 
+        /// <summary>
+        ///     Gets a <typeparamref name="TMessageComponent"/> by the specified <paramref name="customId"/>.
+        /// </summary>
+        /// <typeparam name="TMessageComponent">The type of the component to get.</typeparam>
+        /// <param name="customId">The <see cref="IMessageComponent.CustomId"/> of the component to get.</param>
+        /// <returns>The component of type <typeparamref name="TMessageComponent"/> that was found.</returns>
```

Comment:
```suggestion
        /// <returns>
        ///     The component of type <typeparamref name="TMessageComponent"/> that was found, <see langword="null"> otherwise.
        /// </returns>
```